### PR TITLE
12 home page

### LINKED ISSUE #12: Home page
**Task title**
Home page

**Task description**
Creat the home page, wich containe 6 sections

**Subtasks**

- [x] #22
- [x] #23
- [x] #24
- [x] #25
- [x] #26
- [x] #27

**Screenshots**

1. hero section
hero

2. section 2
section 2

3. section 3
section3

4. section 4
section 4

5. section 5
tickets

6. section 6
section6

**Link to the component on Figma**
https://www.figma.com/file/4OUnD74RgIW6tsUFdb7gcW/Untitled?node-id=2%3A6773&mode=dev

### PR DESCRIPTION
**Pull request by M07am3dM**
**Description**
Create the home page, which contain 6 sections

**Subtasks**

- [x] #22
- [x] #23
- [x] #24
- [x] #25
- [x] #26
- [x] #27 

**Resolve**
Issue Fix #12

### CODE REVIEW

**Comment by dzc0d3r** [comment]
Ooops! there is a typo here.. i guess i forgot to drink coffee lol 🤣 
Since i will be creating a tickets collection to manage the data dynamically it's fine for now.

screenshot


<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Firefox 115.0 on Linux x86_64
Viewport:  1366 x 615 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-57--innerspace.netlify.app%2F&os=OS%20X&os_version=Big%20Sur&browser=Firefox&browser_version=115.0&resolution=1366x615&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-57--innerspace.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=6531af78a0b8da00082f66bf&commentID=6531b6151a31971a7ce2c86f&resolution=resolved)

**Comment by dzc0d3r** [comment]
Maybe we remove the arrows and make the cards bigger .. what do you think @M07am3dM , @Lilicod ?
screenshot


<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Firefox 115.0 on Linux x86_64
Viewport:  1366 x 615 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-57--innerspace.netlify.app%2F&os=OS%20X&os_version=Big%20Sur&browser=Firefox&browser_version=115.0&resolution=1366x615&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-57--innerspace.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=6531af78a0b8da00082f66bf&commentID=6531b67c185e4f1cd54a6e49&resolution=resolved)

**Comment by dzc0d3r** [comment]
Completely forgot about it .. we need to remove it
@M07am3dM 

screenshot


<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Firefox 115.0 on Linux x86_64
Viewport:  1366 x 615 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-57--innerspace.netlify.app%2F&os=OS%20X&os_version=Big%20Sur&browser=Firefox&browser_version=115.0&resolution=1366x615&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-57--innerspace.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=6531af78a0b8da00082f66bf&commentID=6531b6dbc7afe1173f406520&resolution=resolved)

**Comment by M07am3dM** [comment]
I will handle it @dzc0d3r , i already refactor the cards components structure, and look those too.

File: src/pages/index.jsx [hunk 1]
```diff
@@ -2,32 +2,27 @@ import Link from "next/link";
...
+        <Layout>
+            <p>{t("test")}</p>
+            <div style={{ display: "flex", flexDirection: "row", gap: "20px" }}>
+                <Link href='/' locale='en'>
+                    English
+                </Link>
+                <Link href='/' locale='ar'>
+                    العربية
+                </Link>
+            </div>
```
Comment (dzc0d3r) [inline src/pages/index.jsx]: Completely forgot about it ... we need to remove it @M07am3dM

**Comment by dzc0d3r** [comment]
Thank You @M07am3dM

**Review by dzc0d3r — APPROVED** [review]
So far so good .. however we need to open an issue for the translation .. and another for the minor styling updates later on

File: src/pages/index.jsx (hunk 1 above)
Comment (M07am3dM) [reply src/pages/index.jsx]: I deleted the unnecessary and unused code, what about the typo in the tickets @dzc0d3r .

**Review by meryem08 — APPROVED** [review]
Nice Work @M07am3dM and @dzc0d3r 👏 We still need to adjust some style and add the translation.

**Comment by M07am3dM** [comment]
> Nice Work @M07am3dM and @dzc0d3r 👏 We still need to adjust some style and add the translation.

Yes of course, we will do it in separate task.

**Pull request merged**